src/sentry/tasks/post_process.py

```diff
@@ -336,53 +364,52 @@ def post_process_group(
 
             return
 
+        group_states = kwargs.get("group_states")
+
+        if group_states is None:
+            # error issue
+            group_states = {
+                1: {
```

Comment:
I think this should probably be a `Sequence[GroupState]` instead of a `Dict[int, GroupState]`, to match the format the we pass `GroupState` in as.

Reply:
oops, yes